src/libYARP_dev/src/devices/ControlBoardWrapper/ControlBoardWrapper.cpp

```diff
@@ -259,8 +259,7 @@ bool ControlBoardWrapper::checkROSParams(Searchable &config)
         }
         return true;
     }
-    yError() << partName << "should never get here!" << __LINE__;
-    return false;  // should never get here
+    // should never get here
```

Comment:
I'd also remove the `else` al line 166, which is useless since the former `if` contains a `return` so we can cleanup more.